### PR TITLE
Adds Occitan language file

### PR DESCRIPTION
**Pull request by Mejans**
I hope it's fine :)

### CODE REVIEW

File: locales/oc.po [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+msgid "Quality"
+msgstr "Qualitat"
+
+#: yoga_image_optimizer/data/ui/main-window.glade:409
+msgid "JPEG Options"
+msgstr "Opcions JPEG"
+
+#: yoga_image_optimizer/data/ui/main-window.glade:512
+msgid "WebP Options"
+msgstr "Opcion WebP"
```
Comment (flozz) [inline locales/oc.po:72]: "Options" is plural here (like you translated for JPEG and PNG)

**Review by flozz — CHANGES_REQUESTED** [review]
Translations seems mostly ok but I think there are a few mistakes (please tell me if I am wrong, I only spotted some incoherence, I do not speak Occitan ;) )

File: locales/oc.po [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+msgid "Slow PNG optimization"
+msgstr "Optimizacion PNG lenta"
+
+#: yoga_image_optimizer/data/ui/main-window.glade:575
+msgid "PNG Options"
+msgstr "Opcions PNG"
+
+#: yoga_image_optimizer/image_formats.py:48
+msgid "WebP (lossless)"
+msgstr "WebP (pèrda de qualitat)"
```
[the remark this answers is not in the document]
Comment (Mejans) [reply locales/oc.po:84]: oups, you are right!

**Review by flozz — APPROVED** [review]

**Comment by flozz** [comment]
Thank you :)

**Pull request merged**